HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -526,16 +526,20 @@ def self.apply_occupants(model, hpxml, num_occ, cfa, space, schedules_file)
     occ_lost = 1 - occ_lat - occ_conv - occ_rad
 
     # Create schedule
+    people_sch = false
```

Comment:
Please don't mix data types. This will result in it either being a boolean (false) or an object, which is confusing.

Use `people_sch = nil` and then use `if not people_sch.nil?` below. (Though in ruby, I don't think you even need to set `people_sch` to a value, it will default to nil.)